index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
```

Comment:
i feel completely neutral this repository. i feel no particular draw or repulsion from this repository. i wish for nothing and i regret for nothing. i am neither satisfied, nor happy, nor upset about this repository. i feel to this repository like a bee does to a skyscraper. it exists and has no relation to my person. for even if the skyscraper should fall and crash and burn, there is no relation to my hive, and i have no reason to feel any sort of way. i feel like a leaf drifting through the breeze, never being able to land on the ground, never being pulled away from the air. there lies no contempt, no content for everything feels like it lays still as the world moves to a halt.